ArcGISRuntimeSDKQt_QMLSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.qml

```diff
@@ -143,71 +142,26 @@ Rectangle {
                 }
             }
         }
-    }
-
-    // map callout window
-    Rectangle {
-        id: callout
-        width: row.width + (10 * scaleFactor) // add 10 for padding
-        height: 40 * scaleFactor
-        radius: 5
-        border {
-            color: "lightgrey"
-            width: .5
-        }
-        visible: false
 
-        MouseArea {
-            anchors.fill: parent
-            onClicked: mouse.accepted = true
+        calloutData {
+            location: calloutLocation
+            // We use the HTMl to bold, increase the pt size, and centre-align the the damageType title.
+            title: "<br><b><font size=\"+2\">" + damageType + "</font></b>"
```

Comment:
you could refactor to use QML's string.arg() method - 

`title: "<br><b><font size=\"+2\">%1</font></b>".arg(damageType)`